feat: reduce size CSS output file (#266)
openedx/frontend-build: README.rst

3. Install the development version of frontend-build
   ``npm i --save-dev @edx/frontend-build@file:./frontend-build``.

Optimization
-----------
To increase optimization by reducing unused CSS, you can set ``USE_PURGECSS=true`` in ``.env`` or as ENV var in the corresponding MFE.

> However, that would lead to an increase in build time by 30%. It's thus recommended to use this option after finishing with changes of an MFE, not while developing.

Reviewer: ```suggestion
However, note that doing this will increase build time by 30%. It's thus not recommended to use this option during development.
```

> This shall increase performance by 20% as measured by `lighthouse`_. For more information about `optimizing MFEs see this issue`_.

Reviewer: ```suggestion
On the other hand, enabling PurgeCSS will increase browser performance for the end user by as much as 20% (as measured by `lighthouse`_).  Operators are encouraged to do so for production deployments.

For more information about optimizing MFEs, refer to the `issue #138`_ in the wg-frontend repository.
```

.. _lighthouse: https://developer.chrome.com/docs/lighthouse/overview/

> .. _optimizing MFEs see this issue: https://github.com/openedx/wg-frontend/issues/138

Reviewer: ```suggestion
.. issue #138: https://github.com/openedx/wg-frontend/issues/138
```
